### PR TITLE
Update tsc-enum.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Fix broken link

### CODE REVIEW

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by nem035 — APPROVED** [review]

File: typescript/introduction/tsc-basic-types/tsc-enum.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ author: kapnobatai136
 type: normal
 category: must-know
 links:
-  - >-
-    [Enum
-    Type](typescriptlang.org/docs/handbook/basic-types.html#enum){documentation}
+  - '[Enum Type](https://www.typescriptlang.org/#enum){documentation}'
```
Comment (kapnobatai137) [inline typescript/introduction/tsc-basic-types/tsc-enum.md:6]: this link doesn't lead to the `enum` page

```suggestion
  - '[Enum Type](https://www.typescriptlang.org/docs/handbook/basic-types.html#enum){documentation}'
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

File: typescript/introduction/tsc-basic-types/tsc-enum.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply typescript/introduction/tsc-basic-types/tsc-enum.md]: Yeah, I have mistakenly copied the wrong one, thanks!

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by kapnobatai137 — DISMISSED** [review]

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**